src/Storage/Device/CloudflareR2.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+use Utopia\Storage\Storage;
+
+class CloudflareR2 extends S3
+{
+    /**
+     * Regions constants
+     */
+
+    /**
+     * Regions constants
+     */
+    const WNAM = 'us-west-1';
+
+    const ENAM = 'us-east-1';
+
+    const WEUR = 'eu-west-1';
+
+    const EEUR = 'eu-east-1';
+
+    const APAC = 'ap-southeast-1';
+
+    const AUTO = 'auto';
```

Comment:
Can you link to the reference for this?

Reply:
https://developers.cloudflare.com/durable-objects/platform/data-location/       -- I went through this doc

Reply:
Please add the link as a comment in the code.